### PR TITLE
fix: add mock cluster creation to integration test

### PR DESCRIPTION
**Pull request by dnitsch**

### CODE REVIEW

File: .github/workflows/test-ignore.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: CI-ScratchPad-PR
```
Comment (paulholiday) [inline .github/workflows/test-ignore.yml]: Is this still needed or is it just for reference?

Comment (dnitsch) [reply .github/workflows/test-ignore.yml]: that's just for reference but can be removed actually...

File: kubebuilder-controller/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# kubebuilder-controller
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
```
Comment (paulholiday) [inline kubebuilder-controller/README.md:5]: Pls add details

File: kubebuilder-controller/controllers/reststrategy_controller.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	seederv1alpha1 "github.com/dnitsch/reststrategy/kubebuilder-controller/api/v1alpha1"
+	"github.com/dnitsch/reststrategy/seeder"
+	log "github.com/dnitsch/simplelog"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	SuccessMessage = "sucessfully synced resource: %s in namespace: %s"
```
Comment (paulholiday) [inline kubebuilder-controller/controllers/reststrategy_controller.go]: spelling

File: kubebuilder-controller/controllers/reststrategy_controller_test.go [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+	RestStrategyName      = "test-rest-strategy-mocked"
+	RestStrategyNameFail  = "test-rest-strategy-mocked-fail"
+	RestStrategyNamespace = "default"
+
+	timeout  = time.Second * 10
+	duration = time.Second * 10
+	interval = time.Millisecond * 250
+)
+
+func succesEndpoint() http.Handler {
```
Comment (paulholiday) [inline kubebuilder-controller/controllers/reststrategy_controller_test.go]: spelling

**Pull request merged**